.travis.yml

```diff
@@ -1,3 +1,11 @@
 language: ruby
+bundler_args: --without debugging local_testing
 rvm:
-  - 2.1.2
+  - 1.9.3
```

Comment:
The gem says it supports 1.9.3 at the moment.

--
Regards,
Harry Mills

On Tue, Jun 2, 2015 at 3:41 PM, Caius Durling <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/fac/hermod/pull/13#discussion_r31529684>:
>
> >  rvm:
> > -  - 2.1.2
> > +  - 1.9.3
>
> Do we need to support 1.9 anymore? It's dead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fac/hermod/pull/13/files#r31529684>.
>